arch/s390/dfltcc_common.c

```diff
@@ -16,20 +16,31 @@
 static const int PAGE_ALIGN = 0x1000;
 
 void Z_INTERNAL *PREFIX(dfltcc_alloc_window)(PREFIX3(streamp) strm, uInt items, uInt size) {
+    unsigned int n;
     void *p;
     void *w;
 
     /* To simplify freeing, we store the pointer to the allocated buffer right
      * before the window.
      */
-    p = ZALLOC(strm, sizeof(void *) + items * size + PAGE_ALIGN, sizeof(unsigned char));
+    n = items * size;
+    if (n < HB_SIZE)
+      /* DFLTCC always uses HB_SIZE bytes. */
+      n = HB_SIZE;
```

Comment:
Should use `MAX()` macro, it saves 2 lines without sacrificing readability.

Reply:
Done.